### PR TITLE
HBASE-28251 [SFT] Add description for specifying SFT impl during snapshot recovery

### PR DESCRIPTION
**Pull request by wchevreuil**

### CODE REVIEW

File: src/main/asciidoc/_chapters/store_file_tracking.adoc [hunk 1]
```diff
@@ -143,3 +143,21 @@ example, that would be as follows:
 ----
 alter 'my-table', CONFIGURATION => {'hbase.store.file-tracker.impl' => 'FILE'}
 ----
+
+### Specifying trackers during snapshot recovery
+
+It's also possible to specify a given STORE FILE TRACKING implementation when recovering a snapshot
+using the _clone_snasphot_ command. This is useful when recovering old snapshots, taken prior to a
```
Comment (NihalJain) [inline src/main/asciidoc/_chapters/store_file_tracking.adoc:150]: nit: maybe mention the argument name i.e`CLONE_SFT ` here.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 40s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
||| _ Other Tests _ |
|  |   |   1m 26s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5570 |
| Optional Tests |  |
| uname | Linux ad30895da22d 5.4.0-163-generic #180-Ubuntu SMP Tue Sep 5 13:21:23 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 6e421e9d94 |
| Max. process+thread count | 43 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 35s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
||| _ Other Tests _ |
|  |   |   1m 23s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5570 |
| Optional Tests |  |
| uname | Linux f9fde884824d 5.4.0-166-generic #183-Ubuntu SMP Mon Oct 2 11:28:33 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 6e421e9d94 |
| Max. process+thread count | 33 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 55s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 17s |  master passed  |
| +0 :ok: |  refguide  |   3m  5s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotless  |   0m 49s |  branch has no errors when running spotless:check.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 10s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +0 :ok: |  refguide  |   2m 40s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotless  |   0m 48s |  patch has no errors when running spotless:check.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 13s |  The patch does not generate ASF License warnings.  |
|  |   |  17m  2s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5570 |
| Optional Tests | dupname asflicense refguide spotless |
| uname | Linux f8985236123d 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 6e421e9d94 |
| refguide | https://nightlies.apache.org/hbase/HBase-PreCommit-GitHub-PR/PR-5570/1/yetus-general-check/output/branch-site/book.html |
| refguide | https://nightlies.apache.org/hbase/HBase-PreCommit-GitHub-PR/PR-5570/1/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 79 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5570/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by petersomogyi — APPROVED** [review]

**Pull request merged**